easypost/__init__.py

```diff
@@ -997,3 +1001,12 @@ def update(self, **params):
         response, api_key = requestor.request('put', url, params)
         self.refresh_from(response, api_key)
         return self
+
+class Rating(CreateResource):
+    @classmethod
+    def create(cls, api_key=None, **params):
+        requestor = Requestor(api_key)
+        url = "/rating/v1/rates"
+        response, api_key = requestor.request('post', url, params)
+        return convert_to_easypost_object(response, api_key)
+
```

Comment:
there should not be a blank line at the end of the file (just a newline) (PEP8 W391)